app/views/articles/show.html.erb

```diff
@@ -13,14 +13,18 @@
                   class:"btn btn-outline-primary btn-lg" %></li>
     </div>
     <div class="col-4"><br>
-      <li><%= link_to "Edit", edit_article_path(@article),
+      <%if @article.user == current_user%>
```

Comment:
Could we have a method in the controller that does this?

Reply:
Yes, to make the code view more aesthetically pleasing.